Code/Functions/Public/i.ps1

```diff
@@ -0,0 +1,63 @@
+Function i {
+    <#
+        .SYNOPSIS
+
+        Generates a <i> HTML tag.
+        The <a> tag defines a hyperlink, which is used to link from one page to another.
```

Comment:
I guess this is from a copy paste.
You can get the infos here https://www.w3schools.com/tags/tag_i.asp